diff -rup lohit-gujarati-2.4.4/Lohit-Gujarati.sfd lohit-gujarati-2.4.4_mod/Lohit-Gujarati.sfd
--- lohit-gujarati-2.4.4/Lohit-Gujarati.sfd	2009-10-28 17:17:03.000000000 +0530
+++ lohit-gujarati-2.4.4_mod/Lohit-Gujarati.sfd	2011-07-01 16:24:19.119207465 +0530
@@ -20,7 +20,7 @@ OS2Version: 1
 OS2_WeightWidthSlopeOnly: 0
 OS2_UseTypoMetrics: 1
 CreationTime: 1101340800
-ModificationTime: 1256617446
+ModificationTime: 1309517649
 PfmFamily: 17
 TTFWeight: 400
 TTFWidth: 5
@@ -376,7 +376,7 @@ AntiAlias: 1
 FitToEm: 1
 WinInfo: 0 16 13
 AnchorClass2: "Anchor-0"  "'abvm' Above Base Mark in Gujarati lookup 2 subtable" "Anchor-1"  "'blwm' Below Base Mark in Gujarati lookup 1 subtable" "Anchor-2"  "'blwm' Below Base Mark in Gujarati lookup 0 subtable" 
-BeginChars: 65773 345
+BeginChars: 65773 346
 
 StartChar: .notdef
 Encoding: 65536 -1 0
@@ -28165,7 +28165,7 @@ EndChar
 StartChar: numbersign
 Encoding: 35 35 315
 Width: 732
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -28211,7 +28211,7 @@ EndChar
 StartChar: dollar
 Encoding: 36 36 316
 Width: 565
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -28263,7 +28263,7 @@ EndChar
 StartChar: percent
 Encoding: 37 37 317
 Width: 720
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -28319,7 +28319,7 @@ EndChar
 StartChar: ampersand
 Encoding: 38 38 318
 Width: 602
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -28373,7 +28373,7 @@ EndChar
 StartChar: asterisk
 Encoding: 42 42 319
 Width: 535
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -28400,7 +28400,7 @@ EndChar
 StartChar: plus
 Encoding: 43 43 320
 Width: 570
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -28424,7 +28424,7 @@ EndChar
 StartChar: slash
 Encoding: 47 47 321
 Width: 630
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -28440,7 +28440,7 @@ EndChar
 StartChar: zero
 Encoding: 48 48 322
 Width: 479
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -28470,7 +28470,7 @@ EndChar
 StartChar: one
 Encoding: 49 49 323
 Width: 478
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -28490,7 +28490,7 @@ EndChar
 StartChar: two
 Encoding: 50 50 324
 Width: 479
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -28524,7 +28524,7 @@ EndChar
 StartChar: three
 Encoding: 51 51 325
 Width: 479
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -28565,7 +28565,7 @@ EndChar
 StartChar: four
 Encoding: 52 52 326
 Width: 478
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -28593,7 +28593,7 @@ EndChar
 StartChar: five
 Encoding: 53 53 327
 Width: 479
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -28628,7 +28628,7 @@ EndChar
 StartChar: six
 Encoding: 54 54 328
 Width: 479
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -28668,7 +28668,7 @@ EndChar
 StartChar: seven
 Encoding: 55 55 329
 Width: 479
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -28689,7 +28689,7 @@ EndChar
 StartChar: eight
 Encoding: 56 56 330
 Width: 479
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -28737,7 +28737,7 @@ EndChar
 StartChar: nine
 Encoding: 57 57 331
 Width: 478
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -28777,7 +28777,7 @@ EndChar
 StartChar: less
 Encoding: 60 60 332
 Width: 527
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -28796,7 +28796,7 @@ EndChar
 StartChar: equal
 Encoding: 61 61 333
 Width: 480
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -28818,7 +28818,7 @@ EndChar
 StartChar: greater
 Encoding: 62 62 334
 Width: 527
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -28837,7 +28837,7 @@ EndChar
 StartChar: bracketleft
 Encoding: 91 91 335
 Width: 480
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -28857,7 +28857,7 @@ EndChar
 StartChar: backslash
 Encoding: 92 92 336
 Width: 485
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -28873,7 +28873,7 @@ EndChar
 StartChar: bracketright
 Encoding: 93 93 337
 Width: 480
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -28893,7 +28893,7 @@ EndChar
 StartChar: asciicircum
 Encoding: 94 94 338
 Width: 534
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -28912,7 +28912,7 @@ EndChar
 StartChar: braceleft
 Encoding: 123 123 339
 Width: 394
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -28947,7 +28947,7 @@ EndChar
 StartChar: bar
 Encoding: 124 124 340
 Width: 230
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -28963,7 +28963,7 @@ EndChar
 StartChar: braceright
 Encoding: 125 125 341
 Width: 395
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -28998,7 +28998,7 @@ EndChar
 StartChar: asciitilde
 Encoding: 126 126 342
 Width: 572
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -29028,7 +29028,7 @@ EndChar
 StartChar: multiply
 Encoding: 215 215 343
 Width: 477
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -29052,7 +29052,7 @@ EndChar
 StartChar: divide
 Encoding: 247 247 344
 Width: 530
-Flags: HW
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -29084,5 +29084,37 @@ SplineSet
  265 310 l 1,18,19
 EndSplineSet
 EndChar
+
+StartChar: uni20B9
+Encoding: 8377 8377 345
+Width: 608
+Flags: HW
+LayerCount: 2
+Fore
+SplineSet
+354.393 397.203 m 0,0,1
+ 355.295 405.326 355.295 405.326 355.295 409.838 c 1,2,-1
+ 84.545 409.838 l 1,3,-1
+ 108.01 466.696 l 1,4,-1
+ 343.562 466.696 l 1,5,6
+ 338.147 477.525 338.147 477.525 330.025 486.551 c 0,7,8
+ 283.997 536.188 283.997 536.188 180.21 536.188 c 2,9,-1
+ 84.545 536.188 l 1,10,-1
+ 108.01 593.047 l 1,11,-1
+ 557.455 593.047 l 1,12,-1
+ 533.99 536.188 l 1,13,-1
+ 362.516 536.188 l 1,14,15
+ 397.713 515.432 397.713 515.432 410.348 466.696 c 1,16,-1
+ 557.455 466.696 l 1,17,-1
+ 533.99 409.838 l 1,18,-1
+ 418.47 409.838 l 1,19,20
+ 418.47 269.602 418.47 269.602 185.625 252.804 c 1,21,-1
+ 472.865 0.953047 l 1,22,-1
+ 382.615 0.953047 l 1,23,-1
+ 94.4731 250.097 l 1,24,-1
+ 94.4731 303.344 l 1,25,26
+ 343.464 301.538 343.464 301.538 354.393 397.203 c 0,0,1
+EndSplineSet
+EndChar
 EndChars
 EndSplineFont
